src/controllers/breaches.js

```diff
@@ -10,14 +10,16 @@ import { generateToken } from '../utils/csrf.js'
 import { getAllEmailsAndBreaches } from '../utils/breaches.js'
 
 async function breachesPage (req, res) {
-  const emailCount = 1 + (req.user.email_addresses?.length || 0) // +1 because user.email_addresses does not include primary
   // TODO: remove: to test out getBreaches call with JSON returns
   const breachesData = await getAllEmailsAndBreaches(req.user, req.app.locals.breaches)
+  const emailVerifiedCount = breachesData.verifiedEmails?.length ?? 0
```

Comment:
Now we decided to not include unverified emails towards the limit we’d also need to pass the `emailVerifiedCount` to the settings page layout. Currently, we use the [total count for disabling the “Add email address” button](https://github.com/mozilla/blurts-server/blob/MNTOR-1188/breaches-email-count/src/views/partials/settings.js#L130) in the list of monitored emails.

Reply:
I might have lost a thread (or just confused) @flozia – the CTAs for this page behave as though a user has a hard limit regardless of verified/unverified.  For example, if a user has 3 verified emails and 2 unverified emails, they would not be allowed to add another email, and the add-email CTAs are hidden.  So at least in regards to this code, unverified emails _do_ count towards the limit, and I believe this is in line with Settings page?

The difference is with the "3 of 5 emails monitored" copy – this only includes verified emails.  And the CTA below it seems wrongly worded – I'll add you to the UX/PM conversation as well.